src/collections/_documentation/platforms/cocoa/index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Cocoa
+---
+
+{% capture __alert_content -%}
+Version `5.0.0` is still in beta.
+{%- endcapture -%}
+{%- include components/alert.html
+  title="Note"
+  content=__alert_content
+  level="info"
+%}
+
+This is the documentation for our Cocoa SDK (Swift and Objective-C).
+If you are migrating from an older version, please consider our [Migration Guide](https://github.com/getsentry/sentry-cocoa/blob/master/MIGRATION.md). Also always make sure to follow the [changelog](https://github.com/getsentry/sentry-cocoa/blob/master/CHANGELOG.md)
+
+<!-- WIZARD -->
+## Installation {#install}
+
+The SDK can be installed using [CocoaPods](http://cocoapods.org), [Carthage](https://github.com/Carthage/Carthage), or [Swift Package Manager](https://swift.org/package-manager/). This is the recommended SDK for both Swift and Objective-C projects.
+
+It supports iOS (>= 8.0), tvOS (>= 9.0), macOS (>= 10.10) and partially watchOS (>= 2.0). 
+
+### CocoaPods
+
+Installing the SDK with CocoaPods is currently our recommended way.
+To integrate Sentry into your Xcode project, specify it in your _Podfile_:
+
+```ruby
+platform :ios, '8.0'
+use_frameworks! # This is important
+
+target 'YourApp' do
+    pod 'Sentry', :git => 'https://github.com/getsentry/sentry-cocoa.git', :tag => '5.0.0-beta.0'
+end
+```
+<!-- {% sdk_version sentry.cocoa %} -->
+
+Afterwards run `pod install`. In case you encounter problems with dependencies and you are on a newer CocoaPods you might have to run `pod repo update` first.
```

Comment:
Should we recommend running `pod repo update` first, regardless?

Reply:
I removed the whole paragraph.